docs/docsite/rst/network_best_practices_2.5.rst

```diff
@@ -0,0 +1,372 @@
+
+   ansible_connection: network_cli
+   ansible_network_os: eos
+   ansible_user: myuser
+   ansible_ssh_pass: !vault |
+                     $ANSIBLE_VAULT;1.1;AES256
+                     37373735393636643261383066383235363664386633386432343236663533343730353361653735
+                     6131363539383931353931653533356337353539373165320a316465383138636532343463633236
+                     37623064393838353962386262643230303438323065356133373930646331623731656163623333
+                     3431353332343530650a373038366364316135383063356531633066343434623631303166626532
+                     9562
+   ansible_become: yes
+   ansible_become_method: enable
+
+**group_vars/ios.yml**
+
+.. code-block:: yaml
+
+   ansible_connection: network_cli
+   ansible_network_os: ios
+   ansible_user: myiosuser
+   ansible_ssh_pass: !vault |
+                     $ANSIBLE_VAULT;1.1;AES256
+                     34623431313336343132373235313066376238386138316466636437653938623965383732373130
+                     3466363834613161386538393463663861636437653866620a373136356366623765373530633735
+                     34323262363835346637346261653137626539343534643962376139366330626135393365353739
+                     3431373064656165320a333834613461613338626161633733343566666630366133623265303563
+                     8472
+   ansible_become: yes
+   ansible_become_method: enable
+
+**group_vars/vyos.yml**
+
+.. code-block:: yaml
+
+   ansible_connection: network_cli
+   ansible_network_os: vyos
+   ansible_user: myvyosuser
+   ansible_ssh_pass: !vault |
+                     $ANSIBLE_VAULT;1.1;AES256
+                     39336231636137663964343966653162353431333566633762393034646462353062633264303765
+                     6331643066663534383564343537343334633031656538370a333737656236393835383863306466
+                     62633364653238323333633337313163616566383836643030336631333431623631396364663533
+                     3665626431626532630a353564323566316162613432373738333064366130303637616239396438
+                     9853
+
+
+.. FIXME FUTURE Gundalow - Link to network auth & proxy page (to be written)
+
+.. warning:: Never store passwords in plain text
+
+The "Vault" feature of Ansible allows keeping sensitive data such as passwords or keys in encrypted files, rather than as plain text in your playbooks or roles. These vault files can then be distributed or placed in source control. See :doc:`playbooks_vault` for more information.
+
+:ansible_connection:
+
+  The ansible-connection setting tells Ansible how it should connect to a remote device. When working with Ansible Networking, setting this to ``ansible_connection: network_cli`` informs Ansible that the remote node is a network device with a limited execution environment. Without this setting, Ansible would attempt to use ssh to connect to the remote and execute the Python script on the network device, which would fail because Python generally isn't available on network devices.
```

Comment:
Suggest revising to: Ansible uses the ``ansible-connection`` setting to determine how to connect to a remote device. When working with Ansible Networking, set this to netweork_cli so Ansible treats the remote node as a network device with a limited execution environment.